server/src/schema/user/userType.ts

```diff
@@ -116,6 +116,7 @@ builder.prismaObject('User', {
         return prisma.tumiEvent.findMany({
           ...query,
           where: {
+            excludeFromStatistics: false,
```

Comment:
```suggestion
```
This flag was intended to be used for the admin statistics, I would not reuse here.
There could be another one like `hideFromProfile` maybe, I am not sure why it is needed though.